### ISSUE TITLE
How do we perform Naive Bayes Classification for this code ?

### ISSUE THREAD

**Issue by vineal** [callmeal/Flickr4Java#168]
This  code takes in 3 symptoms as input. Next it should classify the symptoms and show the output as the type of doctor that the patient should visit. How do I do that using Naive Bayes Classifier?Please Help :(

`
<?php
		
	include('sessions.php');
?>
<!DOCTYPE html>
<html>
<head>

<title>medi doc</title>
<style>
body{
height: 900px;
    background: -webkit-linear-gradient(white, Lightsteelblue); }
header{
	background-color:#3b5998;
	font-family:Tahoma ;
	color:white;
	font-weight: bold;
	padding:45px;
	}
	#o{position:absolute;
	top:10px;
	left:300px;}
input[name=LogIn]
{
display: block;
position: absolute;
top: 50px;
left: 1300px;
font-size:13px;
color:white;
font-weight:bold;
background-color:#3b5998;
}
#signup
{
position: absolute;
top: 500px;
left: 60px;
font-weight:bold;
font-size:15px;
color:white;
background-color: #006B00;
-webkit-border-radius:8px;

}
#q
{
font-size:27px;
position: absolute;
top: 140px;
left: 830px;
text-align: left;
margin:15px;
padding:65px;

}
#r{
font-size:15px;
position: absolute;
top: 10px;
left: 950px;
color:white;
}
#t{
font-size:15px;
position: absolute;
top: 10px;
left: 1120px;
color:white;
}
#u{
font-family:Tahoma ;
font-size:36px;
position: absolute;
top:90px;
left:900px;
}
#i{
position: absolute;
top:160px;
left:910px;
}
#user_check
{
position: absolute;
top: 240px;
left: 65px;
}
#username_login
{
position:absolute;
top:50px;
left:950px;
}
#password_login{position:absolute;
top:50px;
left:1120px;
}
input[name=name1] , input[name=username] , input[name=password1] , input[name=email1] , input[name=cpassword1]{
-webkit-border-radius: 5px;}
</style>
</head>
<body>
<header><h1 id="o"><b>checklist</b></h1></header>
<form name = "login_form" action = "login.php" method = "POST" accept-charset="UTF-8"  onsubmit="return required()" autocomplete="on" novalidate>
<p id="r">USERNAME </p>
<input type="text" name="username" id= "username_login" size="30" placeholder="USERNAME" style="width:145px" autofocus required autocomplete> 
<p id="t"> PASSWORD</p>
<input type="password" name="password" id = "password_login" size="30" placeholder="PASSWORD" style="width:145px" required >
<p><input type = "submit" name = "LogIn" value = "Log In"></p></div>
</form>

<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
<script type="text/javascript">    // to confirm the password
    function confirmPass() {
        var pass = document.getElementById("password").value
        var confPass = document.getElementById("cpassword").value
        if(pass != confPass) {
            alert('Passwords dont match !');
        }
    }
	</script>
	<script type="text/javascript">    // to check password length
    function validatePass() {
        var pass = document.getElementById("password").value
		
        if(pass.length <8)
         {
            alert('password length is less than 8 characters');
        }
    }
	</script>
	
	<script>    // fill in all fields
	function required()  
{  
var empt = document.forms["myform"]["name1"].value;  
var empt1 = document.forms["myform"]["email1"].value;  
//var empt2 = document.forms["myform2"]["user_check"].value;  
var empt3 = document.forms["myform"]["password1"].value;  
var empt4 = document.forms["myform"]["cpassword1"].value;  
if (empt == "" || empt1 == "" ||empt2 == "" ||empt3 == "" ||empt4 == "" )  
{  
alert("PLEASE FILL ALL THE FIELDS"); 
return false;  
}  
else   
{  
return true;   
}  
}
</script>

<form  name="myform" action = "signup.php" method = "POST" accept-charset="UTF-8" onsubmit="return required()" autocomplete="on" novalidate>
<h2 id="u">enter your symptoms</h2>
<div id="q">
symptom 1 : 
<select name="symptoms">
  <option value="bad taste in mouth" >bad taste in mouth</option>
  <option value="bad breath" >bad breath</option>
  <option value="gap in between teeth" >gap in between teeth</option>
  <option value="chalky white spot on tooth" >chalky white spot on tooth</option>
  <option value="discoloured spot on tooth" >discoloured spot on tooth</option>
  <option value="fracture in tooth" >fracture in tooth</option>
  <option value="sensitive teeth" >sensitive teeth</option>
  <option value="sore throat" >sore throat</option>
  <option value="sores in mouth" >sores in mouth</option>
  <option value="Wisdom tooth problems" >Wisdom tooth problems</option>
  <option value="Tooth decay" >Tooth decay</option>
  <option value="tiny spot on teeth" >tiny spot on teeth</option>
  <option value="toothaches" >toothaches</option>
  <option value="chipped tooth" >chipped tooth</option>
  <option value="nasal swelling" >nasal swelling</option>
  <option value="nasal pain" >nasal pain</option>
  <option value="nasal bleeding" >nasal bleeding</option>
  <option value="blurred vision" >blurred vision</option>
  <option value="difficulty breathing through nose" >difficulty breathing through nose</option>
  <option value="nasal fracture" >nasal fracture</option>
  <option value="Sore throat" >Sore throat</option>
  <option value="faint" >faint</option>
  <option value="difficulty in walking" >difficulty in walking</option>
  <option value="difficulty in speaking" >difficulty in speaking</option>
  <option value="ear sensitivity" >ear sensitivity</option>
  <option value="ear wax" >ear wax</option>
  <option value="common cold" >common cold</option>
  <option value="running nose" >running nose</option>
  <option value="bleeding from nose" >bleeding from nose</option>
  <option value="misshapen nose" >misshapen nose</option>
  <option value="bruising around eyes" >bruising around eyes</option>
  <option value="ear itching" >ear itching</option>
  <option value="hearing loss" >hearing loss</option>
</select>
<br>
symptom 2 : 
<select name="symptoms2">
  <option value="bad taste in mouth" >bad taste in mouth</option>
  <option value="bad breath" >bad breath</option>
  <option value="gap in between teeth" >gap in between teeth</option>
  <option value="chalky white spot on tooth" >chalky white spot on tooth</option>
  <option value="discoloured spot on tooth" >discoloured spot on tooth</option>
  <option value="fracture in tooth" >fracture in tooth</option>
  <option value="sensitive teeth" >sensitive teeth</option>
  <option value="sore throat" >sore throat</option>
  <option value="sores in mouth" >sores in mouth</option>
  <option value="Wisdom tooth problems" >Wisdom tooth problems</option>
  <option value="Tooth decay" >Tooth decay</option>
  <option value="tiny spot on teeth" >tiny spot on teeth</option>
  <option value="toothaches" >toothaches</option>
  <option value="chipped tooth" >chipped tooth</option>
  <option value="nasal swelling" >nasal swelling</option>
  <option value="nasal pain" >nasal pain</option>
  <option value="nasal bleeding" >nasal bleeding</option>
  <option value="blurred vision" >blurred vision</option>
  <option value="difficulty breathing through nose" >difficulty breathing through nose</option>
  <option value="nasal fracture" >nasal fracture</option>
  <option value="Sore throat" >Sore throat</option>
  <option value="faint" >faint</option>
  <option value="difficulty in walking" >difficulty in walking</option>
  <option value="difficulty in speaking" >difficulty in speaking</option>
  <option value="ear sensitivity" >ear sensitivity</option>
  <option value="ear wax" >ear wax</option>
  <option value="common cold" >common cold</option>
  <option value="running nose" >running nose</option>
  <option value="bleeding from nose" >bleeding from nose</option>
  <option value="misshapen nose" >misshapen nose</option>
  <option value="bruising around eyes" >bruising around eyes</option>
  <option value="ear itching" >ear itching</option>
  <option value="hearing loss" >hearing loss</option>
</select><br>
symptom 3 : <select name="symptoms3">
  <option value="bad taste in mouth" >bad taste in mouth</option>
  <option value="bad breath" >bad breath</option>
  <option value="gap in between teeth" >gap in between teeth</option>
  <option value="chalky white spot on tooth" >chalky white spot on tooth</option>
  <option value="discoloured spot on tooth" >discoloured spot on tooth</option>
  <option value="fracture in tooth" >fracture in tooth</option>
  <option value="sensitive teeth" >sensitive teeth</option>
  <option value="sore throat" >sore throat</option>
  <option value="sores in mouth" >sores in mouth</option>
  <option value="Wisdom tooth problems" >Wisdom tooth problems</option>
  <option value="Tooth decay" >Tooth decay</option>
  <option value="tiny spot on teeth" >tiny spot on teeth</option>
  <option value="toothaches" >toothaches</option>
  <option value="chipped tooth" >chipped tooth</option>
  <option value="nasal swelling" >nasal swelling</option>
  <option value="nasal pain" >nasal pain</option>
  <option value="nasal bleeding" >nasal bleeding</option>
  <option value="blurred vision" >blurred vision</option>
  <option value="difficulty breathing through nose" >difficulty breathing through nose</option>
  <option value="nasal fracture" >nasal fracture</option>
  <option value="Sore throat" >Sore throat</option>
  <option value="faint" >faint</option>
  <option value="difficulty in walking" >difficulty in walking</option>
  <option value="difficulty in speaking" >difficulty in speaking</option>
  <option value="ear sensitivity" >ear sensitivity</option>
  <option value="ear wax" >ear wax</option>
  <option value="common cold" >common cold</option>
  <option value="running nose" >running nose</option>
  <option value="bleeding from nose" >bleeding from nose</option>
  <option value="misshapen nose" >misshapen nose</option>
  <option value="bruising around eyes" >bruising around eyes</option>
  <option value="ear itching" >ear itching</option>
  <option value="hearing loss" >hearing loss</option>
</select>

<p><input type = "submit"  name = "Create an account"  id = "signup" style="width:225px ; height:38px" value= "Create an account"></p>

</form>
</div>
</body>
</html>`
